### ISSUE TITLE
Feature List - Due 6/9

### ISSUE THREAD

**Issue by cjc473** [vincentpham123/mumblr#2]
# Feature List

- [x] Should have 7 Features.
    - [x] 3 of those are User Auth, Live Hosting on Render, and Production README.
    - [x] The other 4 are from the [Feature List]([https://open.appacademy.io/learn/swe-campus-hybrid/full-stack-project/Feature-checklist](https://open.appacademy.io/learn/swe-campus-hybrid/full-stack-project/mvp-checklist)) or they have clarified them with you
- [x] Includes two to three detailed bullets on functionality and presentation of feature
- [x] At least one CRUD feature, which states what CRUD operations are planned (creation, reading, updating, deletion)
- [x] Estimates how long it will take the code each Feature
- [x] Correctly formatted
    - [x] Features are listed in an ordered list
    - [x] Each Feature is broken down into bullet points

**Comment by cjc473** [comment]
Looks good so far. A few notes:

1. Add timeline estimate for feature.
2. Mention that users can delete blog posts and likes. This will make blog posts full CRUD and add additional functionality to likes.
3. Following users can be its own feature (a pretty quick one to implement). Add this as a feature to complete before the dashboard, and then make search users your first bonus feature.
4. Add a comments feature. This can be full CRUD, assuming it is on the real Tumblr.
5. Make direct messages a bonus feature. Depending on whether Tumblr DMs are a live chat, you may need to have websockets.

**Comment by vincentpham123** [comment]
Thanks for the corrections Chris. I have implemented the changes
1. For timeline estimates, I have include in the format of (1-2days) depending on the feature
2. I have included that users can delete blog posts, likes, comments, and reblogs. All features are set before the dashboard
3. I have implemented a following users feature right after account creation/login feature and I have made the user search a bonus feature. 
4. Comment feature has been added right after blog posts have been created.
5. Direct messages will be the last bonus feature
